auditbeat/module/auditd/_meta/kibana/7/dashboard/auditbeat-kernel-executions.ndjson

```diff
@@ -1,5 +1,5 @@
+{"attributes":{"description":"","kibanaSavedObjectMeta":{"searchSourceJSON":{"filter":[],"query":{"language":"kuery","query":""}}},"savedSearchRefName":"search_0","title":"Exe Name Tag Cloud [Auditbeat Auditd] ECS","uiStateJSON":{},"version":1,"visState":{"aggs":[{"enabled":true,"id":"1","params":{},"schema":"metric","type":"count"},{"enabled":true,"id":"2","params":{"field":"process.executable","order":"desc","orderBy":"1","size":10},"schema":"segment","type":"terms"}],"params":{"maxFontSize":45,"minFontSize":14,"orientation":"single","palette":{"name":"kibana_palette","type":"palette"},"scale":"linear"},"title":"Exe Name Tag Cloud [Auditbeat Auditd] ECS","type":"tagcloud"}},"coreMigrationVersion":"8.0.0","id":"2efac370-c1ca-11e7-8995-936807a28b16-ecs","migrationVersion":{"visualization":"7.14.0"},"references":[{"id":"d382f5b0-c1c6-11e7-8995-936807a28b16-ecs","name":"search_0","type":"search"}],"type":"visualization","updated_at":"2021-08-04T16:35:57.797Z","version":"WzQ5NjgsMV0="}
+{"attributes":{"description":"Overview of kernel executions","hits":0,"kibanaSavedObjectMeta":{"searchSourceJSON":{"filter":[],"highlightAll":true,"query":{"language":"kuery","query":""},"version":true}},"optionsJSON":{"darkTheme":false,"useMargins":false},"panelsJSON":[{"embeddableConfig":{"enhancements":{}},"gridData":{"h":12,"i":"1","w":16,"x":16,"y":0},"panelIndex":"1","panelRefName":"panel_1","type":"visualization","version":"7.3.0"},{"embeddableConfig":{"enhancements":{}},"gridData":{"h":12,"i":"3","w":16,"x":32,"y":0},"panelIndex":"3","panelRefName":"panel_3","type":"visualization","version":"7.3.0"},{"embeddableConfig":{"enhancements":{}},"gridData":{"h":12,"i":"5","w":16,"x":0,"y":0},"panelIndex":"5","panelRefName":"panel_5","type":"visualization","version":"7.3.0"},{"embeddableConfig":{"enhancements":{}},"gridData":{"h":20,"i":"6","w":48,"x":0,"y":12},"panelIndex":"6","panelRefName":"panel_6","type":"search","version":"7.3.0"}],"timeRestore":false,"title":"[Auditbeat Auditd] Executions ECS","version":1},"coreMigrationVersion":"8.0.0","id":"7de391b0-c1ca-11e7-8995-936807a28b16-ecs","migrationVersion":{"dashboard":"7.14.0"},"references":[{"id":"20a8e8d0-c1c8-11e7-8995-936807a28b16-ecs","name":"1:panel_1","type":"visualization"},{"id":"f81a6de0-c1c1-11e7-8995-936807a28b16-ecs","name":"3:panel_3","type":"visualization"},{"id":"2efac370-c1ca-11e7-8995-936807a28b16-ecs","name":"5:panel_5","type":"visualization"},{"id":"d382f5b0-c1c6-11e7-8995-936807a28b16-ecs","name":"6:panel_6","type":"search"}],"type":"dashboard","updated_at":"2021-08-04T16:35:57.797Z","version":"WzQ5NzAsMV0="}
```

Comment:
Decoding the embeded JSON objects is nice, what do you think about storing them with indentation as they were before?
They wouldn't be strictly NDJSON files... but this would make them to be easier to review. `setup` would take care of fixing the format as expected by the API.

Reply:
So the JSONs would no longer be delimited by a new line. What else would you use instead as a delimiter? My first idea would be putting every asset into a separate file and putting each dashboard into a different folder so we know which asset belongs to each dashboard. If you have anything simpler, maybe returning to JSONs, please let me know.

In general, I do not like the idea but if this helps module developers, I can be persuaded.

Reply:
> So the JSONs would no longer be delimited by a new line. What else would you use instead as a delimiter?

It could still be delimited by a new line, we could decode one object after the other using a json decoder. We would lose though the possibility of continuing to decode objects after one is invalid, but in that sense it would be the same as with dashboards in beats before 7.15.

> My first idea would be putting every asset into a separate file and putting each dashboard into a different folder so we know which asset belongs to each dashboard. If you have anything simpler, maybe returning to JSONs, please let me know.

In integrations every asset is stored in a different file now https://github.com/elastic/integrations/tree/master/packages/apache/kibana, so this could be definitely an option, with the benefit of using the same format in both places.

> In general, I do not like the idea but if this helps module developers, I can be persuaded.

In my opinion it really helps :slightly_smiling_face: Compare the changes in dashboards in these two PRs:
* https://github.com/elastic/beats/pull/27520/files
* https://github.com/elastic/beats/pull/27119/files

Similar changes, but in the second one there are much easier to see.